itest/lnd_mpp_test.go

```diff
@@ -13,10 +13,42 @@ import (
 	"github.com/lightningnetwork/lnd/lntest/node"
 	"github.com/lightningnetwork/lnd/lntest/wait"
 	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
 	"github.com/lightningnetwork/lnd/routing/route"
 	"github.com/stretchr/testify/require"
 )
 
+// testValidateMPPParams validates that a payment with multipath parameters
+// exceeding the allowed maximum amount fails with the expected error.
+func testValidateMPPParams(
+	ht *lntest.HarnessTest, mts *mppTestScenario, paymentAmt btcutil.Amount,
```

Comment:
the format is not quite right, see https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#wrapping-long-function-definitions